README.md

```diff
@@ -1,13 +0,0 @@
-# Google Student Training in Engineering Program
```

Comment:
You should keep or update this README.md instead of deleting it. The readme at the root directory is what shows up on the repo's main page.